bin/dlang/install

```diff
@@ -12,7 +12,7 @@ case "$DMD" in
     dmd*   ) PKG= ;;
     1.*    ) PKG="dmd1=$DMD-$DIST" ;;
     2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;;
-    2.*    ) PKG="dmd-bin=$DMD" ;;
+    2.*    ) PKG="dmd-bin=$DMD libphobos2-dev=$DMD" ;;
     *      ) echo "Unknown \$DMD ($DMD)" >&2; false ;;
```

Comment:
Since `set -e` is used, it will make the script fail, but yeah, a explicit `exit` could be used too.